examples/medplum-provider/src/components/tasks/TaskList.tsx

```diff
@@ -173,10 +173,14 @@ function TaskTitle(props: TaskCellProps): JSX.Element {
       setTitle(<>{props.resource.title}</>);
     } else if (props.resource.resourceType === 'QuestionnaireResponse') {
       fetchQuestionnaireTitle().catch(console.error);
+    } else if (props.task.code) {
+      setTitle(<CodeableConceptDisplay value={props.task.code} />);
     } else {
+      // TODO what should this else actually be?
```

Comment:
Previously, the else block was causing throwing an error since `props.task.code` (an object) is not a valid ReactNode. What would be an appropriate wayto cast a task to a string here if none of the other blocks trigger? Perhaps `Task/${id}`?